cli/packages/gateway/connection.go

```
@@ -0,0 +1,105 @@
+				}
+
+				if _, err = destTarget.Write(bufferedData); err != nil {
+					log.Error().Msgf("Error writing buffered data: %v", err)
+					return
+				}
+			}
+
+			CopyData(conn, destTarget)
+			return
+		case "PING":
+			conn.Write([]byte("PONG"))
```

golangci-lint (1.62.2)

Error return value of `conn.Write` is not checked

(errcheck)